models/recipe.js

```diff
@@ -1,7 +1,15 @@
 const mongoose = require("mongoose");
 
 const Recipe = mongoose.model("recipes", {
-    //...
-})
+    title: String,
+    level: String,
```

Comment:
For level and dishType you could've used an enum to only allow specific options to be entered. It would prevent obscure descriptions and gives guidance on which to use if combined with a dropdown menu for example.